scalableminds/webknossos
# Script to merge volume tracing into on-disk segmentation

### Steps to test:
- create a back up of a dataset with a segmentation layer (mag1)
- volumetrace some, download as zip
- run `python3 main.py my-volumetracing.zip ../../binaryData/Connectomics_Department/ROI2017_wkw_edit_segmentation/segmentation/1/`
- confirm prompt
- output for me looks like 
```
Found 4 tracing files, which will affect 2 segmentation files
Reading segmentation file 1 of 2, bounding box: ([0, 0, 0], [1024, 1024, 1024])...
  Overwriting tracing buckets in memory...
    Overwriting ([512, 480, 480], [32, 32, 32])
    Overwriting ([512, 512, 480], [32, 32, 32])
    Overwriting ([512, 512, 512], [32, 32, 32])
  Writing segmentation file back to disk...
Reading segmentation file 2 of 2, bounding box: ([1024, 0, 0], [1024, 1024, 1024])...
  Overwriting tracing buckets in memory...
    Overwriting ([1024, 512, 480], [32, 32, 32])
  Writing segmentation file back to disk...
Done.
```

### Issues:
- fixes #3343 

------
